quinn/examples/client-async-std.rs

```diff
@@ -0,0 +1,161 @@
+//! This example demonstrates an HTTP client that requests files from a server.
```

Comment:
I'm not sure the fairly trivial differences justify duplicating the maintenance cost of completely duplicating the client/server examples. Maybe a single dedicated, simpler example that just establishes a connection?